### PR TITLE
Add documentation about enhanced ecommerce task

### PR DESCRIPTION
**Pull request by dwhenry**
https://trello.com/c/cuhxDKMB/150-add-documentation-around-enhanced-ecommerce-data-export

### CODE REVIEW

File: source/manual/alerts/enhanced-ecommerce.html.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+owner_slack: "#search-team"
+title: Enhanced ecommerce data export
+parent: "/manual.html"
+layout: manual_layout
+section: Icinga alerts
+last_reviewed_on: 2017-06-27
+review_in: 1 months
+---
+
+This process is related to collection of data for analytics and is run daily so that data can be uploaded to GA for newly created content. This process can be rerun multiple times without side effect.
```
Comment (suzannehamilton) [inline source/manual/alerts/enhanced-ecommerce.html.md]: Just occurred to me: does running it several times fill up the disk? When does the cleanup job run?

Comment (suzannehamilton) [inline source/manual/alerts/enhanced-ecommerce.html.md]: Oops, just saw your comment below. :)

File: source/manual/alerts/enhanced-ecommerce.html.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Ecommerce events only send the `content_id` to google analytics. In order to make the data valuable we upload additional data fields linked to the `content_id` which can then be used for reporting within GA.
+
+This task is responsible for generating the data file on the production server `search-1.api` and deleting historical files - we currently keep the last 10 files on record.
+
+Current the file is left on the server to be manually uploaded into GA.
```
Comment (MatMoore) [inline source/manual/alerts/enhanced-ecommerce.html.md]: Should say "currently"

**Comment by MatMoore** [comment]
Looks good - could we add a link in to some GA docs on query time imports for context?

**Comment by dwhenry** [comment]
@MatMoore added link to GA `query time time` page.

File: source/manual/alerts/enhanced-ecommerce.html.md (hunk 2 above)
Comment (dwhenry) [inline source/manual/alerts/enhanced-ecommerce.html.md]: fixed

**Pull request merged**